src/structures/option.rs

```diff
@@ -37,13 +37,13 @@ pub struct Config {
     #[structopt(long)]
     pub no_preview: bool,
 
-    /// FZF overrides for cheat selection
-    #[structopt(long, env = "NAVI_FZF_OVERRIDES")]
-    pub fzf_overrides: Option<String>,
+    /// finder overrides for cheat selection
+    #[structopt(long, env = "NAVI_FINDER_OVERRIDES")]
```

Comment:
This is a breaking change :(

I think we should keep the old name even if that doesn't make much sense. At least for now

Reply:
Ack. Reverted.